src/ImageCommand.cc

```diff
@@ -164,7 +164,10 @@ int FindImage::construct_protobuf(
     const Json::Value& cmd = jsoncmd[_cmd_name];
 
     Json::Value results = get_value<Json::Value>(cmd, "results");
-    results["list"].append(VDMS_IM_PATH_PROP);
+
+    if (get_value<bool>(results, "return_blob", true)){
```

Comment:
This is called just blob for FindEntity. I like the shorter one but just keep them consistent.

Reply:
Never mind. I think you made them consistent later though FindEntity now has a default of true with return_blob. Two concerns:
- we don't say return_avg or return_sum. So blob might be more suitable when comparing with the other things. Results already implies returning stuff.
- For images, it is natural to expect blobs but not so for entities. It should have a default of false.

Reply:
This is what I suggested, but the agreement was that it was either default for all, or for none, to make it consistent, and since it made sense for images to be the default, we decided to also make it the default for entities as well. 

Also, it make sense to make true as the default for all, as if you add an entity with a blob, you are more likely to be interested in getting the blob when findEntity. 

I will leave it as default=true for all, unless we decide otherwise then.

Reply:
also, I am changing the name to "blob" in all cases.